style.css

```diff
@@ -11,6 +11,9 @@ a:link {
 a:hover {
     color: greenyellow
 }
-/* - normalnie link ma byc czarny
-- po najechaniu (hover) `greenyellow`
-- usunac z niego podkreslenie */
\ No newline at end of file
+
+.animal-image {
+    width: 200px;
```

Comment:
wymaganie biznesowe, bylo takie, zeby obrazki mialy conajmniej 240px. 

poporaw to proszę